blackbox/docs/admin/privileges.rst

```diff
@@ -40,6 +40,13 @@ user is allowed to execute ``SELECT``, ``SHOW``, ``REFRESH``, ``COPY TO``,
 and ``SET SESSION`` statements, as well as using the available user defined
 functions, on the object for which the privilege applies.
 
+.. NOTE::
+
+   DQL privileges only be granted for ``SET SESSION`` statements.
+
+   :ref:`SET GLOBAL <ref-set-desc>` privileges cannot be granted to regular
```

Comment:
```suggestion
   :ref:`SET GLOBAL <ref-set-desc>` privileges cannot be granted because ``SET
   GLOBAL`` statements can only be issued by the superuser.```